index.rst

```diff
@@ -2,24 +2,206 @@
+It should be remembered that there are minimal functionality tests and other categories of tests completed at various stages of the system lifecycle to safeguard the integrity of each component in addition to the tests discussed in this section.
+
+SAL Scripts
+-----------
+
+SAL Scripts are specialized components designed to execute short lived actions in the observatory control system, including, but not limited to; enabling components, preparing the system for calibrations, taking calibration, tracking targets and taking data, etc.
+In principle, most operations executed by the system will be performed using the ScriptQueue, by SAL Scripts.
+
+By default, these components are developed using the BaseScript Python class provided by `salobj`_.
+They are developed in one of the two available repositories; `ts_externalscripts`_ and `ts_standardscripts`_, and each change to the code undergoes a peer review process to ensure TSSW development guidelines have been followed.
+This includes ensuring that sufficient unit tests are present.
+See :tstn:`010` for details on the use case for each of these repositories.
+
+.. _salobj: ts-salobj.lsst.io
+.. _ts_externalscripts: https://github.com/lsst-ts/ts_externalscripts
+.. _ts_standardscripts: https://github.com/lsst-ts/ts_standardscripts
+
+The usage of SAL Scripts for integration testing comes, mainly, in two different varieties:
+
+#.  Scripts used in standard operations that can be executed in an environment where the definition of success can be explicitly defined (e.g. track_target_and_take_image).
+    These can also be used to test fault scenarios by running the script in a mode that we know will cause a particular condition.
+    For example, use the ``track_target_and_take_image`` script in a situation that would cause the system to run into a limit while taking data.
+#. A custom made script to test a particular condition (e.g. forcing a fault condition).
+
+In the first case, we simply use a regular operation (e.g. slew and take data) with a particular configuration for which we can easily predict the outcome, whereas scripts designed to execute an integration test have pre-defined predictable behavior.
+The first line of testing consists in making sure the script executes successfully, followed by a verification that the data produced by the different components matches our expectations (see :ref:`verification-artifacts`).
+
+The most significant advantage of using standard operational SAL Scripts in integration and verification tests is that we are using the same tools that are regularly used in operations.
+Most of these operations exercise the vast majority of the system requirements, which makes them a perfect tool to verify them.
+Since the majority of these uses TSSW high-level libraries (e.g. ts_observatory_control), they are extremely maintainable, requiring minimal to no updates when there are underlying changes to the CSCs behavior.
```

Comment:
Consider:
Furthermore, because the majority of the tests utilize TSSW high-level libraries (e.g. ts_observatory_control), the tests are easily maintained, requiring minimal to no updates when there are underlying changes to the CSCs behavior.